app/services/message_formatter.rb

```diff
@@ -24,6 +25,16 @@ def self.format_message(message)
     MessageFormatter.new(message).format
   end
 
+  def self.extract_mentions(message)
+    MessageFormatter.new(message).mentions
```

Comment:
Because ideally this shouldn't be static methods.  I just did this to be consistent for now.  Eventually the MessageFormatter instance should be the direct interface, not this